plugins/algoltek-aux/algoltek-aux.quirk

```diff
@@ -1,2 +1,2 @@
-[MST\VEN_25A4&DEV_AG9421]
-Plugin = algoltek-aux
+#[MST\VEN_25A4&DEV_AG9421]
+#Plugin = algoltek-aux
```

Comment:
Rather than do this can't @YiJaneWu share the right values for `fu_dpaux_device_get_dpcd_dev_id()`?

Reply:
Currently we don't have a value set in it. Is this necessary to set value for  fu_dpaux_device_get_dpcd_dev_id()?

Reply:
@YiJaneWu it's causing problems for other plugins because they all try to probe in your plugin. For example I see on one system on my end:

```
Loading…                 [*****                                  ]03:24:53.883 FuEngine             no GUIDs for device ZynqMP DP AUX [1e95e9045ae167af37a77a1df296b8d60dd4af7d]
```

Reply:
So; yes if you can still set a value for the DPCD in your firmware, please do so and fix the plugin to use it.

Reply:
OK, we will plan to use OUI or set device ID in our firmware.